src/Sylius/Bundle/ProductBundle/Behat/ProductContext.php

```diff
@@ -289,4 +294,40 @@ public function theFollowingOptionTranslationsExist(TableNode $table)
 
         $manager->flush();
     }
+
+    /**
+     * @param ProductInterface $product
+     * @param array            $data
+     */
+    private function configureProductPricingCalculator(ProductInterface $product, array $data)
+    {
+        $product->getMasterVariant()->setPricingCalculator($data['pricing calculator']);
+
+        if (!isset($data['calculator configuration']) || '' === $data['calculator configuration']) {
+            throw new \InvalidArgumentException('You must set chosen calculator configuration');
+        }
+
+        $product->getMasterVariant()->setPricingConfiguration($this->getPricingCalculatorConfiguration($data));
+    }
+
+    /**
+     * @param array $data
+     *
+     * @return array
+     */
+    private function getPricingCalculatorConfiguration(array $data)
+    {
+        $calculatorConfiguration = $this->getConfiguration($data['calculator configuration']);
+
+        $finalCalculatorConfiguration = array();
+        $channelRepository = $this->getRepository('channel');
+
+        foreach ($calculatorConfiguration as $channelCode => $price) {
+            $channel = $channelRepository->findOneBy(array('code' => $channelCode));
+
+            $finalCalculatorConfiguration[$channel->getId()] = (int) round($price * 100);
```

Comment:
I don't think we need to `round`. There can't be any precision loss when multiplying by `100`.